templates/drawers.mustache

```diff
@@ -98,82 +98,174 @@
-                    <div class="drawer-toggler drawer-left-toggle open-nav d-print-none">
-                        <button
-                            class="btn icon-no-margin"
-                            data-toggler="drawers"
-                            data-action="toggle"
-                            data-target="theme_boost-drawers-courseindex"
-                            data-toggle="tooltip"
-                            data-placement="right"
-                            title="{{#str}}opendrawerindex, core{{/str}}"
-                        >
-                            <span class="sr-only">{{#str}}opendrawerindex, core{{/str}}</span>
-                            {{#pix}} t/index_drawer, moodle {{/pix}}
-                        </button>
-                    </div>
-                {{/courseindex}}
-                {{#hasblocks}}
-                    <div class="drawer-toggler drawer-right-toggle ml-auto d-print-none">
-                        <button
-                            class="btn icon-no-margin"
-                            data-toggler="drawers"
-                            data-action="toggle"
-                            data-target="theme_boost-drawers-blocks"
-                            data-toggle="tooltip"
-                            data-placement="right"
-                            title="{{#str}}opendrawerblocks, core{{/str}}"
-                        >
-                            <span class="sr-only">{{#str}}opendrawerblocks, core{{/str}}</span>
-                            <span class="dir-rtl-hide">{{#pix}}t/blocks_drawer, core{{/pix}}</span>
-                            <span class="dir-ltr-hide">{{#pix}}t/blocks_drawer_rtl, core{{/pix}}</span>
-                        </button>
-                    </div>
-                {{/hasblocks}}
-            </div>
-            {{> theme_boost_union/javascriptdisabledhint }}
-            {{> theme_boost_union/infobanners }}
-            {{{ output.full_header }}}
-            {{#secondarymoremenu}}
-                <div class="secondary-navigation d-print-none">
-                    {{> core/moremenu}}
+
+    {{^user_is_editing}}
+    {{#regions.offcanvas.hasblocks}}
+        {{< theme_boost/drawer }}
+            {{$id}}theme_boost-drawers-offcanvas{{/id}}
+            {{$drawerclasses}}drawer drawer-top {{/drawerclasses}}
+            {{$drawercontent}}
+                {{> theme_boost_union/offcanvas}}
+            {{/drawercontent}}
+            {{$forceopen}}0{{/forceopen}}
+            {{$drawerstate}}show-drawer-offcanvas{{/drawerstate}}
+            {{$tooltipplacement}}top{{/tooltipplacement}}
+            {{$drawercloseonresize}}1{{/drawercloseonresize}}
+            {{$closebuttontext}}{{#str}}closeoffcanvas, theme_boost_union{{/str}}{{/closebuttontext}}
+        {{/ theme_boost/drawer}}
+    {{/regions.offcanvas.hasblocks}}
+    {{/user_is_editing}}
+    
+    <div id="page" data-region="mainpage" data-usertour="scroller" 
```

Comment:
To ease future diffs with upstream, please do not add additional line wraps here.